FE/todo-app/src/components/formComponents/InfoBox.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ComponentPropsWithoutRef } from "react";
+import { cn } from "@/lib/utils";
+
+type Props = ComponentPropsWithoutRef<"p"> & {
+  children: string;
+};
+
+export const InfoBox = (props: Props) => {
```

Comment:
nech mě hádat.. TS problém si vyřešil tak že jsi odmazal ten typ co? :D to co jsi v této funci provedl je přesně důvod proč se má FC používat, udělal jsi totální zvěrstvo s tím childrenem a ten FC ti to určitě říkal :D

Reply:
:D :D :D 
Nn kamo :D  ja jel automat a myslel jen na to ze tam budu davat text a vsude jinde jsem zvykly na children: ReactNode a tu jsem to napral ani nevim proc a FC jsem uplne zapomnel 👯  sry :P